OmsAgent/extension-test/oms_extension_tests.py

```diff
@@ -51,14 +51,27 @@
            '$ python -u oms_extension_tests.py length [image...]'))
 is_long = sys.argv[1] == 'long'
 is_autoupgrade = sys.argv[1] == 'autoupgrade'
+is_instantupgrade = sys.argv[1] == 'instantupgrade'
+
 
 if len(sys.argv) > 2:
-    if sys.argv[2] == 'verbose':
+    if sys.argv[2] == 'debug':
         vms_list = sys.argv[3:]
-        runwith = '--verbose'
+        runwith = '--debug'
+    elif sys.argv[3] == 'debug':
```

Comment:
@jamvar when would debug ever be sys.argv[3]? My understanding of sysargv:
0 = oms_extension_test.py
1 = one of [short, long, autoupgrade, instantupgrade]
2 = either debug, first specified image, or nothing
3 = either first specified image (if debug), second image (no debug) or nothing
4 = either second specified image (if debug), third image (no debug) or nothing
etc.